scripts/globals/trusts/Ayame.lua

```diff
@@ -0,0 +1,121 @@
+-----------------------------------
+--  Trust: Ayame
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/magic")
+require("scripts/globals/utils")
+require("scripts/globals/trusts")
+require("scripts/globals/ability")
+require("scripts/globals/msg")
+require("scripts/globals/magicburst")
+
+function onTrustSpawn(trust)
+    local master = trust:getMaster();
+	
+	trust:addMobMod(dsp.mobMod.SKILL_LIST, dsp.trust.AYAME);
+    trust:addMobMod(dsp.mobMod.SPECIAL_SKILL, dsp.trust.AYAME);
+	trust:setDamage(trust:getMainLvl() * 2);
+	trust:setMod(dsp.mod.STR, trust:getMaxSkillLevel(trust:getMainLvl(), dsp.job.SAM, dsp.skill.GREAT_KATANA));
+	trust:setMod(dsp.mod.DEX, trust:getMaxSkillLevel(trust:getMainLvl(), dsp.job.SAM, dsp.skill.GREAT_KATANA));
+	trust:setMod(dsp.mod.VIT, trust:getMaxSkillLevel(trust:getMainLvl(), dsp.job.SAM, dsp.skill.GREAT_KATANA));
+	trust:setMod(dsp.mod.AGI, trust:getMaxSkillLevel(trust:getMainLvl(), dsp.job.SAM, dsp.skill.GREAT_KATANA));
+	trust:setMod(dsp.mod.INT, trust:getMaxSkillLevel(trust:getMainLvl(), dsp.job.SAM, dsp.skill.GREAT_KATANA));
+	trust:setMod(dsp.mod.MND, trust:getMaxSkillLevel(trust:getMainLvl(), dsp.job.SAM, dsp.skill.GREAT_KATANA));
+	trust:setMod(dsp.mod.CHR, trust:getMaxSkillLevel(trust:getMainLvl(), dsp.job.SAM, dsp.skill.GREAT_KATANA));
+	trust:setMod(dsp.mod.ATT, trust:getMaxSkillLevel(trust:getMainLvl(), dsp.job.SAM, dsp.skill.GREAT_KATANA));
+	trust:setMod(dsp.mod.MATT, trust:getMaxSkillLevel(trust:getMainLvl(), dsp.job.SAM, dsp.skill.GREAT_KATANA));
+	trust:setMod(dsp.mod.ACC, trust:getMaxSkillLevel(trust:getMainLvl(), dsp.job.SAM, dsp.skill.GREAT_KATANA));
+	trust:setMod(dsp.mod.EVA, trust:getMaxSkillLevel(trust:getMainLvl(), dsp.job.SAM, dsp.skill.GREAT_KATANA));
+	trust:setMod(dsp.mod.DEF, trust:getMaxSkillLevel(trust:getMainLvl(), dsp.job.SAM, dsp.skill.GREAT_KATANA));
+	trust:setMod(dsp.mod.MDEF, trust:getMaxSkillLevel(trust:getMainLvl(), dsp.job.SAM, dsp.skill.GREAT_KATANA));
+	
+
+	if master:hasTrust(dsp.trust.NAJI) then
+	    trust:PrintToArea("I am interested to see how you spar, Naji. Just let your breath be used for exertion, not mindless chatter.", dsp.msg.channel.PARTY, dsp.msg.area.PARTY, trust:getName());
```

Comment:
Oh, addendum because I noticed this on my first skim but failed to put it in initial review. Sometimes you're using your `SendPartyMessage` function, but other times like here you're using `PrintToArea`. Make sure all occurrences of trust party chat stay consistent...!